src/platform/site-wide/sass/modules/_m-homepage-blog.scss

```diff
@@ -0,0 +1,17 @@
+.homepage-blog {
+  &__image {
+    height: 100%;
+
+    img {
+      width: 100%;
+      height: 350px;
```

Comment:
I'm curious where `350px` comes from. I can certainly argue that it works very well, but I'm just not sure I see it in the design. I'm also not sure I'm looking at the most up-to-date mobile design (or have the most up-to-date understanding of discussion around possible updates).

Reply:
For the best responsive behavior, we need to set a fixed height. I grabbed this height from the Sketch mockup (which could be outdated).